### ISSUE TITLE
authorizedCallbackWithCode$ no longer throws error on invalid url

### ISSUE THREAD

**Issue by timo-haas** [damienbod/angular-auth-oidc-client#516]
authorizedCallbackWithCode$ no longer throws error on invalid url

With Version [10.0.10](https://github.com/damienbod/angular-auth-oidc-client/commit/229cdbbcb2bbe8e4a2bd1364f0ffbdbd237dbab2) OidcSecurityService. authorizedCallbackWithCode$
no  longer throws error on invalid url. 

Is there an alternative way to validate the url?

**Comment by damienbod** [comment]
@timo-haas 

Yes, I had to revert this because it was throwing exceptions in normal running after a refresh, or a logout, login. The signin can happen either with this function, or per silent renew. I reverted this so that the standard way works again. The URL is not always the callback URL with the code and state and so should not throw, but only handle this if it is the callback URL. 

The URL is validated later using the specs. Does this make sense?

Greetings Damien

**Comment by timo-haas** [comment]
My bad. I did not realize an empty subscribe() will throw an error.
Fixed in #518
